subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/report/PerformanceFlakinessDataProvider.java

```diff
@@ -56,6 +56,24 @@
      */
     BigDecimal getFailureThreshold(String scenario);
 
+    ScenarioRegressionResult getScenarioRegressionResult(String scenarioName, double regressionPercentage);
+
+    enum ScenarioRegressionResult {
+        STABLE_REGRESSION(true),
+        SMALL_FLAKY_REGRESSION(false),
+        BIG_FLAKY_REGRESSION(true);
+
+        private final boolean failsBuild;
```

Comment:
Seems like I don't see where `failsBuild` is used?

Reply:
Now I do.